packages/cache/src/internal/tar.ts

```diff
@@ -14,14 +14,20 @@ async function getTarPath(
   switch (process.platform) {
     case 'win32': {
       const systemTar = `${process.env['windir']}\\System32\\tar.exe`
-      if (compressionMethod !== CompressionMethod.Gzip) {
+      const gnuTar = `${process.env['PROGRAMFILES']}\\Git\\usr\\bin\\tar.exe`
+      if (existsSync(gnuTar)) {
```

Comment:
Just curious, why does call to `existsSync` here does not have a `fs.` prefix but is present in the above `isGnuTarInstalled()` function?

Reply:
Due to the way it is imported

Reply:
Again, just nitpicking here. I wanted to know should we make this consistent? Not sure about conventions in the javascript world, but it is considered good practice to avoid blanket `*` imports.

Reply:
Looks like it is not a problem otherwise there should have been a caution in docs. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#namespace_import